scot/datasets.py

```diff
@@ -0,0 +1,85 @@
+# Released under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+# Copyright (c) 2016 SCoT Development Team
+
+from os import makedirs
+from os.path import expanduser, isfile, isdir, join
+from requests import get
+import numpy as np
+
+from .matfiles import loadmat
+from .eegtopo.eegpos3d import positions
+from . import config
+
+
+datadir = expanduser(config.get("scot", "data"))
+valid_datasets = ["mi"]
+files = {"mi": ["motorimagery.mat"]}
+url = {"mi": "https://github.com/scot-dev/scot-data/raw/master/scotdata/"}
+
+
+def fetch(dataset="mi", datadir=datadir):
+    """Fetch example dataset.
+
+    If the requested dataset is not found in the location specified by
+    `datadir`, the function attempts to download it.
+
+    Parameters
+    ----------
+    dataset : str
+        Which dataset to load. Currently only 'mi' is supported.
+    datadir : str
+        Path to the storage location of example datasets. Datasets are
+        downloaded to this location if they cannot be found. If the directory
+        does not exist it is created.
+
+    Returns
+    -------
+        data : list of dicts
+            The data set is stored in a list, where each list element
+            corresponds to data from one subject. Each list element is a
+            dictionary with the following keys:
+              "eeg" ... EEG signals
+              "triggers" ... Trigger latencies
+              "labels" ... Class labels
+              "fs" ... Sample rate
+              "locations" ... Channel locations
+    """
+    if dataset not in valid_datasets:
+        raise ValueError("Example data '{}' not available.".format(dataset))
+    if not isdir(datadir):
+        makedirs(datadir)
+
+    data = []
+
+    for file in files[dataset]:
```

Comment:
In python 2 `file` is a built-in function. It's probably bad karma to overwrite it :)